torch/csrc/jit/passes/frozen_conv_folding.cpp

```diff
@@ -88,7 +88,7 @@ bool FoldFrozenConvBatchnorm(Block* b) {
         at::ScalarType bias_dtype = bn_rm.scalar_type();
         at::ScalarType weight_dtype = conv_w.scalar_type();
         at::DeviceType weight_device = conv_w.device().type();
-        if (weight_device == at::kCUDA &&
+        if ((weight_device == at::kCUDA || weight_device == at::kCPU) &&
```

Comment:
instead could we just remove the device type check here?

Reply:
@davidberard98 Thanks for the comment. Remove the device type check. Please help to take a look again.